pkgs/tools/misc/fastfetch/default.nix

```diff
@@ -0,0 +1,128 @@
+{ fetchFromGitHub
+, lib
+, nix-update-script
+, stdenv
+, testers
+, #nativeBuildInputs
```

Comment:
This is not a tutorial on how to write your first  expression. Remove these useless comments.